.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,40 @@
+<!--
+Thanks for your interest in the project. Bugs filed and PRs submitted are appreciated!
+
+Please make sure that you are familiar with and follow the [Code of Conduct](https://allcontributors.org/docs/en/project/code-of-conduct) for
+this project.
```

Comment:
It seems there's an extra new line here.

Reply:
Didn't see that one. It wasn't intentional 👎

Reply:
Okay, no worries.